deploy/laa-manage-your-civil-cases/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
I am confiused by these notes. (For my benefit) would it be possibl to add some more explanation of what they are trying to achieve?

Reply:
these notes are used by helm to output a message on successful deploy:
https://github.com/ministryofjustice/laa-manage-your-civil-cases/actions/runs/14596980269/job/40945305994
if you have a look at `Deploy using Helm to UAT` section you will see where this is used.

But essentially this is an automated file created by helm and I havent added ro removed anything. Some teams like to make this output service specific, but as long as the URL is provided I dont think it matters too much.